en-US/ranking/criteria.md

```diff
@@ -2,117 +2,218 @@
+- If a song has been edited to have higher tempo, `(Sped Up Ver.)` marker must be used at the end of the title. If the title contains similar marker by default, it should be replaced with `(Sped Up Ver.)`.
+- If a song has been edited to have lower tempo, `(Slowed Down Ver.)` marker must be used at the end of the title. If the title contains similar marker by default, it should be replaced with `(Slowed Down Ver.)`.
+- Special unicode characters must be filtered to their nearest standard equivalent. Stars such as `★ ☆` should be substituted with an asterisk (`*`). Other special characters should be romanized or left out on a case-by-case basis.
 
-##### Difficulty Names & Description
+# Media
 
-The standard names for difficulties in a mapset are Beginner, Easy, Normal, Hard, Insane, and Expert.
-Should you choose not to use the standard names, the difficulty names in your set must show clear sign of progression in difficulty consistent with that of the mapset and should be easy to interpret.
+## Rules
 
-The following descriptions are what each difficulty should typically look like. Keep in mind that this isn't concrete, and there is room for subjectivity in individual cases. It is only to serve a general outlook on how each map should look and feel.
+- **The total file size of the mapset must not exceed 50 MB.**
 
-* **Beginner**
-     * For players who are being exposed to rhythm games for the first time.
-     * Notes should typically be on every other beat in most cases and once every beat to portray emphasis in the music.
-     * Jumps are suitable for this difficulty when emphasizing important sounds in the music.
-     * Scroll velocity changes must not be used at this difficulty level for any other purposes than scroll velocity normalization.
+## Background
 
-* **Easy**
-     * This difficulty can be thought of as an "upgraded Beginner."
-     * Players at this level will be able to play at slightly higher speeds and execute more complicated patterns.
-     * One beat streams and 1/2 bursts are generally comfortable to these players.
-     * Jumps can be used more freely but sparingly, as players at this level may still have trouble with them.
-     * Scroll velocity changes must not be used at this difficulty level for any other purposes than scroll velocity normalization.
+### Rules
 
-* **Normal**
-     * Maps at this level should typically start branching into patterns that don't undermap the song as heavily compared to easy and beginner.
-     * 1/2 rhythms with more complicated note placements and jumps are suitable for this difficulty.
+- **Every map in the mapset must have a background image.** Using different background images for different maps is acceptable. A mapset does not require a banner image.
```

Comment:
Move the "A mapset does not require a banner image." to guidelines as "While not required, it is encouraged to include a banner image for your mapset."